.hound.yml

```diff
@@ -0,0 +1,2 @@
+ruby:
+  config_file: .rubocop.yml
```

Comment:
files missing this config
https://github.com/iTechArt-RubyLab/Ruby-Lab/blob/rubylab-2021-2022/.hound.yml#L1

withour this line PR check will pass green even if hound find something